app/controllers/api/v1/classifications_controller.rb

```diff
@@ -36,10 +36,23 @@ def build_resource_for_update(update_params)
     classification
   end
 
+  def sms_ids_from_links(link_params)
+    sms = if ids = link_params.delete(:set_member_subjects)
+      SetMemberSubject.select(:id).find(ids)
```

Comment:
Cool I'll leave as is then. We can always revisit if this proves to be a bottleneck.